extension/src/extension.ts

```diff
@@ -37,16 +39,16 @@ export function createExtension(): ExasolExtension {
         findInstallations(_context: Context, metadata: ExaMetadata): Installation[] {
             return findInstallations(metadata.allScripts.rows);
         },
-        findInstances(_context: Context, _installation: Installation): Instance[] {
-            return [];
+        findInstances(context: Context, version: string): Instance[] {
+            return findInstances(context);
```

Comment:
Version needs to be evaluated in future, right?

Reply:
Yes, correct. For now we only support a single version.
We could throw an error when someone tries to a wrong version, but that's not necessary I guess.